shared/utils/api/utils.go

```diff
@@ -91,3 +94,41 @@ func IsTransactionDue(rp *rocketpool.RocketPool, startTime time.Time) (bool, tim
 	return isDue, timeUntilDue, nil
 
 }
+
+//  Expects a 129 byte 0x-prefixed EIP-712 signature and returns v/r/s as v uint8 and r, s *[32]byte
+
+func ParseEIP712(signature string) (uint8, *[32]byte, *[32]byte, error) {
```

Comment:
can we return a pointer to a struct with fields for v, r, s instead of returning a single scalar and 3 pointers